fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1185,6 +1185,10 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true, comment = "whether to collect metrics for materialized view by default")
     public static boolean enable_materialized_view_metrics_collect = true;
 
+    @ConfField(mutable = true, comment = "whether to enable text based rewrite by default, if true it will " +
+               "build ast tree in materialized view initialization")
+    public static boolean enable_materialized_view_text_based_rewrite = true;
```

Comment:
why use a config not a SessionVariable?

Reply:
then session variable is enough? In which case we should use config other than session variable of enable_materialized_view_text_match_rewrite?